site/docs/v1/tech/core-concepts/integration-points.adoc

```diff
@@ -62,6 +63,50 @@ When using OAuth/OIDC, there are multiple kinds of tokens: link:/docs/v1/tech/oa
 
 These tokens can be consumed by APIs or other systems to verify that the holder of the token has been authorized by FusionAuth. link:/learn/expert-advice/tokens/[Learn more about JWTs].
 
+=== JWT Storage
+
+FusionAuth recommends the Authorization Code grant, where there is a server side component which exchanges the one-time use authorization code for an access token. This server side component offers a lot of flexibility when it comes to storing the JWT.
+
+But what should you do with it? After all, an access token is a bearer token, which can't be revoked easily (though see link:/learn/expert-advice/tokens/revoking-jwts[here for strategies]).
+
+This depends on what you are using the token for and your security requirements. The FusionAuth team has outlined a link:/learn/expert-advice/authentication/login-authentication-workflows[number of common flows here].
+
+The primary storage mechanisms the FusionAuth team recommends are:
+
+* sending the token down as a `Secure`, `HTTPOnly` cookie
+* storing it server side, in a session
+
+.JWT Storage Options
+[cols="3,1,1"]
+|===
+| Feature | Cookie | Server Side Session 
+
+|Scales horizontally well
+|Yes
+|No
+
+|Sticky sessions required
+|No
+|Yes
+
+|Transparently sent on requests
+|Yes
+|N/A
+
+|Token can be presented to servers on domains other than that which the server side component resides in
+|No
+|Yes
```

Comment:
If you use a cookie to access a server side session, how is this different than the jwt in a cookie in this context?

Reply:
if you have a SPA living on example.com, and you want to access an API living on foo.com, you can't if you send the JWT down as a cookie, because it will never be sent there.

If you are using a session, the SPA can request example.com/fooAPI, which can proxy the request to foo.com.

Reply:
That is the scenario I was trying to document here.